spec/factories/email_templates.rb

```diff
@@ -0,0 +1,26 @@
+# == Schema Information
+#
+# Table name: email_templates
+#
+#  id               :integer          not null, primary key
+#  status           :integer(enum)    not null
+#  hide_recipients  :boolean          not null
+#  subject          :string           not null
+#  content          :string           not null
+#
+FactoryGirl.define do
+  factory :email_template do
+    status :accepted
+    hide_recipients false
+    subject "Accepted EmailTemplate-Subject"
+    content "Accepted EmailTemplate-Content"
+  end
+
+  factory :email_template_accepted, parent: :email_template do
+    status :accepted
+  end
+
+  factory :email_template_rejected, parent: :email_template do
```

Comment:
Nevermind, when i commented the default values in :email_template were like "Subject accepted" and "content accepted", so it either changed or i looked at the wrong diff (happened already twice today..). Those values are fine